tpp/log4net.config

```diff
@@ -0,0 +1,11 @@
+<log4net>
```

Comment:
Just wanted to double that this file is required for all core, user, tpp?

Reply:
Done.Removed from all core, user, tpp